app/src/main/java/com/example/myapplication/ui/ProductAttributeComposable.kt

```diff
@@ -0,0 +1,169 @@
+import com.example.myapplication.R
+import com.example.myapplication.model.ButtonAction
+import com.example.myapplication.model.ProductAttributeData
+import com.example.myapplication.ui.theme.PriceBackground
+import com.example.myapplication.ui.theme.PrimaryColor
+import com.example.myapplication.ui.theme.Shapes
+import com.example.myapplication.ui.theme.Typography
+import com.kole.myapplication.cms.nnsettings.NNSettingsString
+
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+@Composable
+fun ProductAttributeComposable(
+    modifier: Modifier = Modifier,
+    productAttributeData: ProductAttributeData,
+    onButtonClicked: (ButtonAction) -> Unit
+) {
+    Scaffold(
+        topBar = {
+            TopAppBar(backgroundColor = PrimaryColor, title = {
+                Column(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.Center
+                ) {
+                    Text(
+                        text = productAttributeData.name, style = Typography.h6
+                    )
+                }
+            }, actions = {
+                Box(
+                    modifier = Modifier
+                        .clip(CircleShape)
+                        .background(Color.DarkGray)
+                        .size(32.dp)
+                ) {
+                    Icon(imageVector = Icons.Default.Close,
+                        contentDescription = null,
+                        tint = Color.White,
+                        modifier = Modifier
+                            .align(Alignment.Center)
+                            .clickable {
+                                onButtonClicked(
+                                    ButtonAction.NavigateButton(
+                                        NavigationDestination.MainScreen()
+                                    )
+                                )
+                            })
+                }
+            })
+        }, backgroundColor = PrimaryColor
+    ) {
+        Column(modifier = Modifier.padding(16.dp, 32.dp)) {
+            Text(text = productAttributeData.title, style = Typography.h5)
+            Text(
+                text = productAttributeData.description, modifier = Modifier.padding(0.dp, 16.dp)
+            )
+            val prodAttrName = productAttributeData.name.lowercase()
+            Text(
+                text = NNSettingsString(
+                    "SimilarAttrProducts",
+                    stringResource(R.string.similar_attr_products, prodAttrName),
+                    mapOf(Pair("{ATTR_NAME}", prodAttrName)),
+                ), style = Typography.h6, modifier = Modifier.padding(0.dp, 16.dp)
+            )
+            SimilarProducts(productAttributeData)
+        }
+    }
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+private fun SimilarProducts(productAttributeData: ProductAttributeData) {
```

Comment:
We need some ui tweak here. Try to set image height smaller (it was around 130dp and fill the parents width)
Can we remove the space between product title and coins pill?
Can we make coins pill height a bit smaller?